src/server/game/Weather/Weather.cpp

```diff
@@ -28,10 +28,11 @@
 #include "ScriptMgr.h"
 #include "Util.h"
 #include "World.h"
```

Comment:
Is this still needed? (also put Map.h in its sorted place)

Reply:
> Is this still needed? (also put Map.h in its sorted place)

Doesn't look like it's needed, no. I need to fix the build on GCC anyways apparently, forgot how touchy it can be.

Reply:
There is still one use, setting the timer from the world config.